torch/jit/frontend.py

```diff
@@ -142,27 +145,34 @@ def get_jit_class_def(cls, self_name):
     method_defs = [get_jit_def(method[1],
                    self_name=self_name) for method in methods]
 
-    source = dedent(inspect.getsource(cls))
-    py_ast = ast.parse(source)
-    ctx = SourceContext(source, False)
+    sourcelines, file_lineno = inspect.getsourcelines(cls)
+    source = ''.join(sourcelines)
+    filename = inspect.getsourcefile(cls)
+    py_ast = ast.parse(dedent(source))
+    leading_whitespace_len = len(source.split('\n', 1)) - len(source.split('\n', 1))
```

Comment:
`len(source.split('\n', 1)) - len(source.split('\n', 1))`? Doesn't seem right. Is this tested?